PainteraShowContainer fails when opening multiscale but not painteraData raw data

```
Exception in Application start method
Exception in thread "main" java.lang.reflect.InvocationTargetException
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at sun.launcher.LauncherHelper$FXHelper.main(LauncherHelper.java:767)
Caused by: java.lang.RuntimeException: Exception in Application start method
        at com.sun.javafx.application.LauncherImpl.launchApplication1(LauncherImpl.java:917)
        at com.sun.javafx.application.LauncherImpl.lambda$launchApplication$1(LauncherImpl.java:182)
        at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException
        at org.janelia.saalfeldlab.paintera.PainteraShowContainer.isLabelData(PainteraShowContainer.java:322)
        at org.janelia.saalfeldlab.paintera.PainteraShowContainer.start(PainteraShowContainer.java:173)
        at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$8(LauncherImpl.java:863)
        at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$7(PlatformImpl.java:326)
        at com.sun.javafx.application.PlatformImpl.lambda$null$5(PlatformImpl.java:295)
        at java.security.AccessController.doPrivileged(Native Method)
        at com.sun.javafx.application.PlatformImpl.lambda$runLater$6(PlatformImpl.java:294)
        at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
        at com.sun.glass.ui.gtk.GtkApplication._runLoop(Native Method)
        at com.sun.glass.ui.gtk.GtkApplication.lambda$null$5(GtkApplication.java:139)
```

Fix should be easy, just check the multi scale case in `PainteraShowContainer.isLabelData` as well.

--- a/src/main/java/org/janelia/saalfeldlab/paintera/PainteraShowContainer.java
+++ b/src/main/java/org/janelia/saalfeldlab/paintera/PainteraShowContainer.java
@@ -314,11 +314,16 @@ public class PainteraShowContainer extends Application {
 	/* *************************************************** */
 
 	private static boolean isLabelData(N5Reader reader, String group) throws IOException {
+
 		if (N5Helpers.isPainteraDataset(reader, group)) {
 			JsonObject painteraInfo = reader.getAttribute(group, N5Helpers.PAINTERA_DATA_KEY, JsonObject.class);
 			LOG.debug("Got paintera info {} for group {}", painteraInfo, group);
 			return painteraInfo.get("type").getAsString().equals("label");
 		}
+
+		if (N5Helpers.isMultiScale(reader, group))
+			return N5Types.isLabelMultisetType(reader, group) || isLabelData(reader, N5Helpers.getFinestLevelJoinWithGroup(reader, group));
+
 		return N5Types.isLabelMultisetType(reader, group) || reader.getDatasetAttributes(group).getDataType().equals(DataType.UINT64);
 	}
 

